### PR TITLE
Now supports 'width' option for select renderer

### PR DESCRIPTION
**Pull request by cstolli**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [ ] #patch# - backwards-compatible bug fix
- [x] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG
* Now supports 'width' option for select renderer

### CODE REVIEW

**Comment by agonza40** [comment]
👍

File: tests/dummy/app/pods/view/renderers/documentation/select.md [hunk 1]
```diff
@@ -170,8 +188,6 @@ Sometimes, it is necessary populate the list with static options which are then
 }
 ```
 
-If you use this option, it will replace the list of options sourced from `enum` but `enum` is still required for validation.
```
Comment (cstolli) [inline tests/dummy/app/pods/view/renderers/documentation/select.md]: pretty sure this was an accident, checking

Comment (cstolli) [inline tests/dummy/app/pods/view/renderers/documentation/select.md]: looks like it just got moved

**Comment by agonza40** [comment]
👍

**Pull request merged**